Dido's project/contact.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Contact Form</title>
+    <link rel="stylesheet" href="./StyleContact.css" />
+  </head>
+  <body class="background">
+    <main>
+      <div>
+        <header>
+          <h1 class="H1">Welcome to our contact form.</h1>
+          <p class="P1">
+            Please submit your contact details and a short detailed request
+            below.
+          </p>
+        </header>
+        <div>
+          <form class="Form">
+            <textarea placeholder="* Your Name Here"></textarea>
```

Comment:
You should make sure that you use the appropriate input type in your form. As you see here, you use a `textarea`, which is widely used for longer text - comments e.t.c. In this file the types that you are looking for are: 
`<input type="text">`
`<input type="email">`
*`<input type="number">` - you can use `<input type="text">` for your phone number input. Later on, when we use JavaScript, we will make sure to check if it is a valid phone number.